lib/pip.js

```diff
@@ -249,6 +249,16 @@ function installRequirements(targetFolder, serverless, options) {
         `${process.getuid()}:${process.getgid()}`,
         '/var/task'
       ]);
+    } else {
+      // Use same user so --cache-dir works
+      dockerCmd.push('-u', getDockerUid(bindPath));
+    }
+
+    for (let path of options.dockerExtraFiles) {
+      pipCmds.push(['cp', path, '/var/task/']);
+    }
+
+    if (process.platform === 'linux') {
```

Comment:
I'm not sure merge conflicts were resolved correctly 🤔

Reply:
The code below this line only applies to Linux. But the `cp` applies to all environments. What do you believe is wrong, exactly?

Reply:
Nothing in particular, the diff just seemed a little odd for what you were changing, but that's probably just github's diff algo